src/alpaca_eval/leaderboards/data_AlpacaEval/alpaca_eval_gpt4_leaderboard.csv

```diff
@@ -15,12 +15,13 @@ vicuna-13b-v1.3,82.11180124223603,1.348769957803504,660,143,2,805,verified,1132
 openbuddy-llama-30b-v7.1,81.54613466334165,1.370658000946423,654,148,0,802,community,968
 llama-2-13b-chat-hf,81.09452736318407,1.3817573087734825,652,152,0,804,minimal,1513
 openchat-13b,80.8695652173913,1.3843738653129234,650,153,2,805,community,1632
-openbuddy-falcon-40b-v9,80.69738480697384,1.3908517976873223,647,154,2,803,community,1089
+openbuddy-falcon-40b-v9,80.69738480697384,1.3908517976873225,647,154,2,803,community,1089
```

Comment:
better leave this line as it is

Reply:
yeap changed it already, idk how it got modified 😅